packages/octopus-oas/schemas/effects/background-blur.yaml

```diff
@@ -0,0 +1,25 @@
+description: Background blur effect.
+
+type: object
+
+allOf:
+  - $ref: '../schemas.yaml#/EffectBase'
+
+  - type: object
+    properties:
+      type:
+        type: string
+        enum:
+          - BACKGROUND_BLUR
```

Comment:
I think what you want is already defined as BOUNDED_BLUR. But, honestly, I am not 100% sure since this was invented by Jan Roztocil and Viktor Chlumsky (not working for Ceros anymore). I'd rather ask @janroztocil if he remembers what we invented for background blur.

Reply:
Sup, I don't anything about `BOUNDED_BLUR` you guys must have added that after I left. However, `BACKGROUND_BLUR` is something we had since forever, it's exists to cover the behaviour of Photoshop layer (background) blur effect which has some weird properties, I don't know from the top of my head but it's bounds could be one of them, could be like it's bounded to the nearest layer group or even document not necessarily the layer itself... so unless you later added `BOUNDED_BLUR` as some kind of abstraction I'd say these 2 are different effects.

And another PoV, this PR exists because apparently something (some octopus converter) produces Octopus files with this property, so even if these 2 were exactly the same the specs needs to cover that. So this PR makes sense to me.

Ideally, we'd want to actually understand the exact difference between the effects and put it into the description. But I'm not sure we have resources to do that.